packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -247,12 +201,16 @@ const AccountSwitcher = props => {
     // Real accounts is always the first tab index based on design
     const isRealAccountTab = active_tab_index === 0;
 
+    const getDemoMT5 = () => {
```

Comment:
I added this function to the client store.
You can connect to it using `demo_mt5_accounts: client.demo_mt5_accounts,`
Please just remove it and use the new one

Reply:
Okay, thanks, will do refactor for this.